src/libcollections/vec.rs

```diff
@@ -1089,6 +1092,9 @@ impl<T> Vec<T> {
 
     /// Clears the vector, removing all values.
     ///
+    /// Note that this method has no effect on the allocated capacity
+    /// of the vector, and is equivalent to calling `truncate(0)`.
```

Comment:
Could this comment be a little less specific about the implementation? As it stands, the implementation is in the comment itself. (I am being a little paranoid here.)